### DISCUSSION TITLE
Proposing a possible checklist for reviewing a pull requst

### DISCUSSION THREAD

**Discussion by ashenafykebede** [lab-antwerp-1/agile-group-4#15] [General]
#### The pull request:
- [ ] has descriptive title
- [ ] has appropriate labels and milestones for easy identification
- [ ] it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the [project board](https://github.com/lab-antwerp-1/agile-group-4/projects/1)
- [ ] has short description (optional but recommended)
- [ ] has an issue linked with it
- [ ] teammates feedback are addressed (if any and if it is very appropriate feedback.)
- [ ] all CI checks pass
- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] all conflicts are resolved (if any)

## Markdown
- [ ] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] all links and images work

### HTML

- [ ] the code is well-formatted
- [ ] the HTML code passes validation
- [ ] there are no inline styles (example: `style='color: red;'`)
- [ ] there are no `<style>` tags with CSS, all styles are hrefs
- [ ] `classes` not `ids` are used  for CSS
- [ ] semantic tags are used
- [ ] spelling and grammar is correct in all site content
- [ ] all links work as expected

### CSS

- [ ] the code is well-formatted
- [ ] passes all the linting checks

---------------------------------------------
Additional comments here (if any)

**Comment by ashenafykebede** [comment]
Dear teammates ,

Here is a possible checklist for reviewing a pull request. You can use it as a guide line for reviewing a PR. 

You can copy the markdown file and paste it in the comment section of the PR which you are reviewing. Then check /uncheck the boxes :wink: 

let me see what you think .

**Reply by omerjava** [reply]
looks very good checklist

**Comment by colevandersWands** [comment]
>  the PR has a change only in one file

this one is unproductive, you'll often want changes in more than one file.  If a PR is contributing a header than it _should_ change the HTML & CSS.

A better checkbox might be "the PR contributes only one focused change"

**Reply by ashenafykebede** [reply]
Thank you so much @colevandersWands for the feedback.

**Reply by colevandersWands** [reply]
your general checks are very complete, don't be surprised if the templates look more like this in the future ; )

**Reply by ashenafykebede** [reply]
😀
